src/lib/client.ts

```diff
@@ -91,6 +96,15 @@ export default class CommonClient extends EventEmitter
         this.current_reconnects = 0
         this.generate_request_id = generate_request_id || (() => ++this.rpc_id)
 
+        if (!dataPack)
+        {
+            this.dataPack = new DefaultDataPack()
```

Comment:
Since this is the sole line in this conditional, please remove the curly braces to preserve some space. Applies to the rest of the PR.